src/GitPrompt.ps1

```diff
@@ -97,6 +97,8 @@ $global:GitPromptSettings = [pscustomobject]@{
 
     EnableWindowTitle                           = 'posh~git ~ '
 
+    AnsiConsole                                 = $Host.UI.SupportsVirtualTerminal -or ($Env:ConEmuANSI -eq "ON")
```

Comment:
I believe the SupportsVirtualTerminal property was added in PS 5.1.  I have started a prototype for ANSI stuff based on v5 classes. In it, I have a PromptRenderer class that has a HostSupportsAnsi method.  That method returns $false for version less than 5.1.  It returns false for Windows version less than 10.0.10586.0 (which is the build that support conhost ANSI seqs).  Finally, it returns the value of $Host.UI.SupportsVirtualTerminal.